javawrapper/device/samples/direct call of wrapped functions/Main.java

```diff
@@ -0,0 +1,164 @@
+package javaWrapper;
+
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+import com.sun.jna.ptr.IntByReference;
+
+import javaWrapper.Iothub_client_wrapperLibrary.AMQP_Protocol;
+import javaWrapper.Iothub_client_wrapperLibrary.HTTP_Protocol;
+import javaWrapper.Iothub_client_wrapperLibrary.IOTHUB_CLIENT_HANDLE;
+import javaWrapper.Iothub_client_wrapperLibrary.IOTHUB_CLIENT_RESULT;
+import javaWrapper.Iothub_client_wrapperLibrary.IOTHUB_MESSAGE_HANDLE;
+import javaWrapper.Iothub_client_wrapperLibrary.IotHubEventCallback;
+import javaWrapper.Iothub_client_wrapperLibrary.IotHubMessageCallback;
+import javaWrapper.Iothub_client_wrapperLibrary.MAP_HANDLE;
+
+
+
+
+
+public class Main {
+
+	public static int callbackCounter = 0;
+	
+	public static IOTHUB_CLIENT_HANDLE handle = new IOTHUB_CLIENT_HANDLE(null);
+	public static IOTHUB_MESSAGE_HANDLE messageHandle;
+	public static int IOTHUB_CLIENT_OK = 0;
+	static final int MESSAGE_COUNT = 5;
+	public static EVENT_INSTANCE[] messages = new EVENT_INSTANCE[MESSAGE_COUNT];
+	static Pointer receiveContext;
+	
+	
+	public static Iothub_client_wrapperLibrary IoTHub = Iothub_client_wrapperLibrary.INSTANCE;
```

Comment:
Should not have to do that in the sample. Should be able to call the Client constructor which instantiates this underneath.